### ISSUE TITLE
Feature: Weight Tracking

### ISSUE THREAD

**Issue by rgarber11** [simonoppowa/OpenNutriTracker#38]
For many people using this app, a place to store a weight measurement each day would also be a good additional piece of data.

**Comment by lapo-luchini** [comment]
Yes, I'd like either that or reading it from Google Fit data directly.

**Comment by vetinari** [comment]
@rgarber11 what about https://github.com/oliexdev/openScale?

**Comment by jordan-lee-code** [comment]
Hi @rgarber11 — confirming this is still outstanding on `develop`.

Code location: `lib/core/data/dbo/user_dbo.dart:16` has only a single `weightKG` field; there's no per-day weight log. Adding this means a new `WeightLogDBO` (pick the next free Hive `typeId`), a simple add-entry UI, plotting in the Profile screen at `lib/features/profile/profile_page.dart`, and round-tripping through the export ZIP.

Labelled `enhancement`.

**Comment by jordan-lee-code** [comment]
Merged in #400. Thanks for filing — this is now on `develop` and will ship with the next release.

If you spot something still wrong on a fresh build, please reopen this issue or open a new one with the details and I'll have a look.
